src/app/child-dev-project/notes/note-details/note-details.component.ts

```diff
@@ -17,12 +19,26 @@ export class NoteDetailsComponent implements ShowsEntity<Note> {
   @Input() entity: Note;
   @ViewChild("dialogForm", { static: true }) formDialogWrapper;
 
+  readonly Child: EntityConstructor<Child> = Child;
+  readonly User: EntityConstructor<User> = User;
+
   INTERACTION_TYPE_CONFIG = INTERACTION_TYPE_CONFIG_ID;
+  includeInactiveChildren: boolean = true;
```

Comment:
```suggestion
  includeInactiveChildren: boolean = false;
```
inactive should be excluded by default